test/hotspot/gtest/gtestMain.cpp

```diff
@@ -278,7 +278,7 @@ static void runUnitTestsInner(int argc, char** argv) {
     int ret;
     if ((ret = init_jvm(argc, argv, is_vmassert_test, &jvm)) != 0) {
       fprintf(stderr, "ERROR: JNI_CreateJavaVM failed: %d\n", ret);
-      abort();
+      os::abort();
```

Comment:
No this should be the native abort function. We failed to create the VM so we don't/can't use any VM service.